package/Dockerfile.submariner-libreswan-git

```
@@ -0,0 +1,40 @@
+FROM fedora:32
+
+RUN dnf -y install --nodocs git audit-libs-devel bison curl-devel fipscheck-devel flex \
+    gcc ldns-devel libcap-ng-devel libevent-devel \
+    libseccomp-devel libselinux-devel make nspr-devel nss-devel \
+    pam-devel pkgconfig systemd-devel unbound-devel xmlto rpm-build \
+    openldap-devel hostname nss-tools
```

Please don’t hard-code build-dependencies. Instead,

```suggestion
RUN dnf -y install --nodocs git-core 'dnf-command(builddep)' nss-tools rpm-build
```

and see the corresponding suggestions below.

`git` pulls in a ton of dependencies, we only need `git-core`. The build-dependencies don’t mention `nss-tools` but it is necessary (https://github.com/libreswan/libreswan/pull/355).